### ISSUE TITLE
VB clustering

### ISSUE THREAD

**Issue by Vuducbao913** [BUTSpeechFIT/VBx#25]
I'm research about VB clustering, but i don't understand its idea. How can it cluster speaker after using ahc. Can you summarize its main idea for me? Thank you

**Comment by fnlandini** [comment]
Hello, 
AHC is used as initialization. That means, obtain an initial assignment of embeddings to speaker labels. The model then iteratively estimates speaker models and reassigns the embeddings to speakers until convergence. I recommend you to take a look at the [publication](https://authors.elsevier.com/a/1dN0k39HpSeKMh) related to this code to have a better grasp on the idea.

**Comment by fnlandini** [comment]
Closing issue due to inactivity. Feel free to reopen it.
